tools/scylla-sstable.cc

```diff
@@ -3249,9 +3258,26 @@ Validate the specified sstables:
         cache_tracker tracker;
         sstables::directory_semaphore dir_sem(1);
         abort_source abort;
-        sstables::sstables_manager sst_man("scylla_sstable", large_data_handler, dbcfg, feature_service, tracker,
-            memory::stats().total_memory(), dir_sem,
-            [host_id = locator::host_id::create_random_id()] { return host_id; }, abort);
+
+        sharded<sstables::storage_manager> sstm;
```

Comment:
scylla-sstable runs on a single-shard, no need for `sharded<>` here.

Reply:
you need sharded since the `storage_manager` is a `peering_sharded_service` and someone has to set the `container` on it
Am I missing something?